jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -59,6 +59,7 @@
           v-model.number="item.value"
           type="number"
           :disabled="!is_editable"
+          :suffix="item.unit"
```

Comment:
unfortunately it seems that unicode characters aren't supported for &#8491; (`&#8491;`), but do we want to swap out with the shorter "A" still?

```suggestion
          :suffix="item.unit.replace('Angstrom', 'A')"
```

Reply:
Before/after:

<div>
<img width="200" alt="Screen Shot 2023-05-15 at 2 18 46 PM" src="https://github.com/spacetelescope/jdaviz/assets/877591/c5f9fed0-2840-40c5-bd25-d060c9880254">

<img width="200" alt="Screen Shot 2023-05-15 at 2 18 28 PM" src="https://github.com/spacetelescope/jdaviz/assets/877591/62ac90f8-eb4c-46b0-853d-d673531e8270">
</div>

Reply:
I honestly wasn't even sure I liked the styling using `suffix` rather than having a separate unit column, but if everyone else thinks the suffix way looks alright I'm fine with this abbreviation.

Reply:
This nice thing about suffix is you don't need to care about how many columns to give it, so I actually like the way you have it (and would be in favor of changing other places to match, even if it means losing support for &#8491;).